src/warnet/client.py

```diff
@@ -14,17 +15,18 @@
 
 def get_bitcoin_debug_log(network: str, index: int) -> str:
     warnet = Warnet.from_network(network)
-    tank = warnet.tanks[index]
-    subdir = "/" if tank.bitcoin_network == "main" else f"{tank.bitcoin_network}/"
-    data, stat = tank.container.get_archive(f"/home/bitcoin/.bitcoin/{subdir}debug.log")
-    out = ""
-    for chunk in data:
-        out += chunk.decode()
-    # slice off tar archive header
-    out = out[512:]
-    # slice off end padding
-    out = out[: stat["size"]]
-    return out
+    docker = warnet.docker
+    container_name = warnet.tanks[index].container_name
+    now = datetime.utcnow()
+
+    logs = docker.api.logs(
+        container=container_name,
+        stdout=True,
+        stderr=True,
+        stream=False,
+        until=now,
```

Comment:
Ooh theres other good options for this: https://docker-py.readthedocs.io/en/stable/containers.html#docker.models.containers.Container.logs

like `since=` and `timestamps=False` (since bitcoin core prints timestamps already?)